src/frontend/src/components/public/PIAFormTabs/collectionUseAndDisclosure/index.tsx

```diff
@@ -90,6 +90,20 @@ const PIACollectionUseAndDisclosure = ({
         'collectionUseAndDisclosure',
       );
     }
+
+    // Disabled editor buttons if the user does not have the MPORole.
+    if (isMPORole() === false) {
+      // If the current user does not have the MPORole,
+      // select button elements inside #collectionNoticeMPO .w-md-editor-toolbar li.
+      const buttons = document.querySelectorAll<HTMLButtonElement>(
+        '#collectionNoticeMPO .w-md-editor-toolbar li > button',
+      );
+
+      // Iterate over all button elements and set the 'disabled' attribute to true.
+      buttons.forEach((button) => {
+        button.disabled = true;
```

Comment:
is there not a disabled parameter we can pass in md-editor?

Reply:
> is there not a disabled parameter we can pass in md-editor?

no, currently if it is the preview mode (disabled to type) it disables the toolbar options on the left like bold, Italic bts but not the buttons on the right(Edit button,Live code, Preview Code etc) so I have written a code to disable it